bears/general/LicenseCheckBear.py

```diff
@@ -0,0 +1,37 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='licensecheck',
+        output_format='regex',
+        output_regex=r'(?P<message>\S+.*)')
+class LicenseCheckBear:
+    """
+    Attempts to determine the license that applies to each file passed to
```

Comment:
I thought our objective for this kind-of bear was to determine whether a file contains a licence or not.
So, use HiddenResults maybe so other bears in future can use it.
Also can we have it do something like it checks a directory for a licence file ? or other bear which depends on it would do that..